src/main/java/io/conduit/SparkDestinationStream.java

```diff
@@ -107,6 +107,12 @@ private void doWrite(Record rec) {
         }
     }
 
+    @SneakyThrows
+    private void updateRecord(Record rec) {
+        deleteRecord(rec);
+        insertRecord(rec);
```

Comment:
Can we somehow ensure this is run in a transaction? For example, the delete may go through but the update may fail.

Reply:
@hariso the readme is updated